fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -434,7 +434,9 @@ public void replayCreateDb(CreateDbInfo createDbInfo) {
         try {
             Database db = new Database(createDbInfo.getId(), createDbInfo.getDbName());
             unprotectCreateDb(db);
-            stateMgr.getStorageVolumeMgr().replayBindDbToStorageVolume(createDbInfo.getStorageVolumeId(), db.getId());
+            if (createDbInfo.getStorageVolumeId() != null) {
```

Comment:
add comment to explain in which situation, the volumeId can be null.

Reply:
done